finite_difference/src/grid_mod.f90

```diff
@@ -67,9 +67,8 @@ module grid_mod
      !! This is the key quantity that determines the region that
      !! is actually simulated. However, we also support the
      !! specification of a model consisting entirely of wet points
-     !! with periodic boundary conditions. Since this does not
-     !! require a T-mask, we do not allocate this array for that
-     !! case.
+     !! Since this does not require a T-mask, we do not allocate
```

Comment:
Please revert/refine in light of the fact that one day we *might* support PBCs.

Reply:
Docs updated.